src/parser/content-evaluator-module.ts

```diff
@@ -90,23 +130,21 @@ export class ContentEvaluatorModule implements Module {
       });
 
       const rawResponse = String(response.choices[0].message.content);
-      const parsedResponse = JSON.parse(rawResponse) as number[];
-      return parsedResponse.map((o) => new Decimal(o));
+      return JSON.parse(rawResponse) as { [k: string]: number };
```

Comment:
it might be a good idea to validate the response with typebox in order to avoid AI mistakes

Reply:
Can you provide an example of what types of mistakes typebox can help avoid?

Reply:
For example one time I made a prompt that instructed it to output an object with value which is a number but the output was a number in string `"1"` instead of just a number, or it could for some reason put a number that is higher than 1. 
I know this is rare but I still like to validate any external input to avoid bugs later in the code.

Reply:
resolved by using typebox